components/calendar/calendar.js

```diff
@@ -575,13 +575,12 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 	updated(changedProperties) {
 		super.updated(changedProperties);
 
-		changedProperties.forEach((oldVal, prop) => {
+		changedProperties.forEach(async(oldVal, prop) => {
```

Comment:
I don't know if this is okay? The `this.updateComplete` was added due to needing the dates to be ready before checking what the focus date should be

Reply:
I think it should be fine... it just means the callbacks won't run sequentially which probably doesn't matter.